nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/InvokeHTTP.java

```diff
@@ -876,10 +851,6 @@ public void onTrigger(ProcessContext context, ProcessSession session) throws Pro
                 int statusCode = responseHttp.code();
                 String statusMessage = responseHttp.message();
 
-                if (statusCode == 0) {
-                    throw new IllegalStateException("Status code unknown, connection hasn't been attempted.");
-                }
-
```

Comment:
Not sure why but I imagine this was here for some reasons. Do we think this is safe to remove?

Reply:
The OkHttp client Response Builder class sets the default value to `-1` so a value of `0` should not be possible.  Perhaps this reflected an issue with earlier versions of the OkHttp client, but with the version 4 series, this should not happen.